plugins/caddy/Caddyfile

```diff
@@ -1,9 +1,15 @@
 # See https://caddyserver.com/docs/caddyfile for more details
+{
+	admin 0.0.0.0:2020
+	auto_https disable_certs
+	http_port 8800
+	https_port 4443
+}
```

Comment:
What's this for?

Reply:
This is to change the default caddy http `80` and https port `443` port to be non privileged port `8800` and `4443`